Python/Product/Analysis/LanguageServer/Structures.cs

```diff
@@ -657,6 +657,10 @@ public struct CompletionList {
         /// The expression that members are being displayed for.
         /// </summary>
         public string _expr;
+        /// <summary>
+        /// When true, completions should commit by default.
```

Comment:
What does `null` mean?

Reply:
Up to the IDE in that case - this is kind-of an option in VS, but in some cases we want to force it to false (e.g. override completions).

I'll clarify the docs and make it null most of the time.